tools/virtualization/installing-libvirt-and-virt-install-on-fedora-linux.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Installing libvirt and virt-install on Fedora Linux  
+subsection: virtualization  
+order: 3  
+---
+
+# Installing libvirt and virt-install on Fedora Linux
+
+1. Execute the following command to install libvirt on the host device, if not already installed.  
+   ```
+   sudo dnf install libvirt -y
+   ```
+   ![01](https://user-images.githubusercontent.com/49605954/131217762-da304d01-eaee-4072-8555-4de0f9ea39f2.png)
```

Comment:
Please let's rather include the image into this repository.

Reply:
*and all other images as well

Reply:
Ah yes. On it.

Reply:
Addressed here https://github.com/developer-portal/content/pull/394/commits/f5287ba3f507704d7d1e046ec7fc10eec1419af7